src/SDKs/Logic/Management.Logic/Microsoft.Azure.Management.Logic.csproj

```diff
@@ -5,7 +5,7 @@
 
 	<PropertyGroup>
 		<Description>Microsoft Azure LogicApps Management Library</Description>
-		<Version>3.1.0</Version>
+		<Version>4.0.0</Version>
 		<PackageId>Microsoft.Azure.Management.Logic</PackageId>
 		<PackageTags>Microsoft Azure LogicApps management;LogicApps;LogicApps management;</PackageTags>
 		<PackageReleaseNotes/>
```

Comment:
Please add `PackageReleaseNotes` here

Reply:
Added, please let me know if the formatting is incorrect